### DISCUSSION TITLE
Command to pay claim blocks to others

### DISCUSSION THREAD

**Discussion by bipedd** [GriefPrevention/GriefPrevention#1831] [Ideas]
I'd really like to see a command such as /blocks pay [amount] [player] added to GriefPrevention so players could give each other blocks. It would basically take the amount of claim blocks entered from the person executing the command and giving it to the player entered.

**Comment by xSepticSidx** [comment]
This can be achieved with some cooperation from your server admin.

A command block chain, fired by a button can interrogate the players claims, take from them and then give them to the player stood in a specific spot. (as an example).
